tasks/main.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_list: "{{ capra_lvm_filesystem_packages }}"
+  tags: [capra_lvm_filesystem_prerequisites]
+
+- name: "Configure default mountpoint directory"
+  file:
+    path: "{{ capra_lvm_filesystem_parent_path }}"
+    state: directory
+    owner: "{{ capra_lvm_filesystem_owner }}"
+    group: "{{ capra_lvm_filesystem_group }}"
+    mode: 0755
+    seuser: unconfined_u
+    serole: object_r
+    setype: default_t
+    selevel: s0
+  tags: [capra_lvm_filesystem_prerequisites]
+
+- vars:
+    _capra_lvm_filesystem_path: "{{ item.path | default(capra_lvm_filesystem_parent_path ~ '/' ~ item.name) }}"
```

Comment:
"Local" variables in our other code use `r_` as a prefix (from `register`, even though they're not always registered variables).